nengo_dl/tensor_graph.py

```diff
@@ -1230,16 +1230,18 @@ def special_set(s, op):
                 # reshape view
                 self.signals[sig] = self.signals[sig.base].reshape(sig.shape)
             else:
+                # slice view
+
                 if sig.shape[1:] != sig.base.shape[1:]:
                     # TODO: support this?
                     raise NotImplementedError("Slicing on axes > 0 is not supported")
```

Comment:
I don't think it would be too hard to do this, and with the code below getting more complicated, it might actually be clearer just to implement it. That is, unless there's some underlying reason TensorFlow can't support it?

Doesn't have to happen in this PR, but something to keep in mind.

Reply:
If there were any other situations where this functionality was used then I'd probably look into implementing this at that point, but as long as it's only used in SimPES it seems easier to just special-case it there.